data_structures/src/serializers.rs

```diff
@@ -158,6 +172,36 @@ impl TryFrom<Vec<u8>> for Message {
                     })
                 })
                 .ok_or(""),
+            FlatBuffersCommand::GetBlocks => message
+                .command_as_get_blocks()
+                .map(|get_blocks| {
+                    let hash_prev_block = match get_blocks
+                        .highest_block_checkpoint()
+                        .unwrap()
```

Comment:
This `unwrap()` can be avoided if the related part of the schema is required (see comment below).